packages/fs-detectors/src/monorepos/get-monorepo-default-settings.ts

```diff
@@ -66,7 +66,7 @@ export async function getMonorepoDefaultSettings(
 
     return {
       monorepoManager: 'turbo',
-      buildCommand: `cd ${relativeToRoot} && npx turbo run build --filter=${projectName}...`,
+      buildCommand: `cd ${relativeToRoot} && npx turbo run build --filter={${projectPath}}...`,
```

Comment:
Are the nested brackets a typo?

Reply:
https://turbo.build/repo/docs/core-concepts/monorepos/filtering#filter-by-directory